src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,20 @@
+const ApiError = require('../exeptions/apiError.js');
+
+const errorMiddleware = (error, req, res, next) => {
+  if (error instanceof ApiError) {
+    res.status(error.status).send({
+      message: error.message,
+      errors: error.errors,
+    });
+  }
```

Comment:
Consider adding a `return` statement after sending the response for `ApiError` to prevent the execution from continuing and potentially sending another response.